.openshift-ci/jobs/push-images/registry_rw_login.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+registry_rw_login() {
+    if [[ "$#" -ne 1 ]]; then
+        die "missing arg. usage: registry_rw_login <registry>"
+    fi
+
+    local registry="$1"
+
+    case "$registry" in
+        docker.io/stackrox)
+            if [[ -z "$DOCKER_IO_PUSH_USERNAME" ]]; then
+                echo "DOCKER_IO_PUSH_USERNAME is not defined"
+                exit 0 # Change to exit 1 later
+            fi
+            docker login --username "$DOCKER_IO_PUSH_USERNAME" --password-stdin docker.io <<< "$DOCKER_IO_PUSH_PASSWORD"
+            ;;
```

Comment:
I think we can remove this block, we shouldn't be using dockerhub anywhere in the repo anymore.

Reply:
Done